apps/etterlatte-behandling/src/main/kotlin/behandling/omregning/OmregningService.kt

```diff
@@ -39,11 +41,21 @@ class OmregningService(
             }
 
             behandlingService.hentAapenOmregning(request.sakId)?.let {
+                val oppgave = oppgaveService.hentOppgaverForReferanse(it.id.toString()).single()
```

Comment:
ops minner om at denne kaster exceptions som ikke eksplisitt håndteres her om det ikke finnes en oppgave som gir ufullstendige feilmeldinger

Reply:
Ja dere kan jo se i oppgaveService for inspo men noe ala det bør være ok.
`            hentOppgaverForReferanse(referanse)
                .filter { it.type == type }
                .singleOrNull { it.erAttestering() }
`